web-ui/src/main/resources/catalog/components/admin/registry/RegistryDirective.js

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (C) 2001-2016 Food and Agriculture Organization of the
+ * United Nations (FAO-UN), United Nations World Food Programme (WFP)
+ * and United Nations Environment Programme (UNEP)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or (at
+ * your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ *
+ * Contact: Jeroen Ticheler - FAO - Viale delle Terme di Caracalla 2,
+ * Rome - Italy. email: geonetwork@osgeo.org
+ */
+
+(function() {
+  goog.provide('gn_registry_directive');
+
+  var module = angular.module('gn_registry_directive', []);
+
+  /**
+     * Provide a Registry element selector.
+   *
+     * Usage:
+     * <div data-gn-registry-browser="registryUrl"></div>
+     */
+  module.directive('gnRegistryBrowser', [
+    'gnLangs', 'gnRegistryService', '$rootScope', '$translate',
+    function(gnLangs, gnRegistryService, $rootScope, $translate) {
+      return {
+        restrict: 'A',
+        replace: true,
+        // scope: {},
+        templateUrl: '../../catalog/components/admin/registry/partials/' +
+            'registrybrowser.html',
+        link: function(scope, element, attrs) {
+          scope.itemClass = [];
+          scope.languages = [];
+          scope.itemCollection = [];
+          scope.item = [];
+          scope.selectedClass = null;
+          scope.selectedCollection = null;
+          scope.selectedLanguages = {};
+          scope.loadingCollection = false;
+
+          scope.registryUrl = '';
+          scope.defaultRegistry = 'http://inspire.ec.europa.eu';
```

Comment:
No, it is the default in case you don't have your own registry. See the default button in the UI
![image](https://user-images.githubusercontent.com/1701393/39625265-f8231b84-4f9c-11e8-9331-a804bb77b855.png)

Reply:
Perfect, thanks for the screenshot to clarify it.